src/posix/platform/openthread-core-posix-config.h

```diff
@@ -118,4 +118,14 @@
 
 #endif
 
+/**
+ * @def OPENTHREAD_CONFIG_CLI_UART_TX_BUFFER_SIZE
+ *
+ * The size of CLI message buffer in bytes.
+ *
+ */
+#ifndef OPENTHREAD_CONFIG_CLI_UART_TX_BUFFER_SIZE
+#define OPENTHREAD_CONFIG_CLI_UART_TX_BUFFER_SIZE 8192
```

Comment:
do we really need such large TX buffer?

Reply:
On a POSIX environment, is there a practical downside of an 8 kB buffer?

1024 bytes was not enough even for modest networks. Some of the COI output depends on how many children you have attached.

Reply:
Can we utilize the flush api when buffer is full?

Reply:
We would need to implement `otPlatUartFlush()` to support that:
https://github.com/openthread/openthread/blob/b49ee0879ac1c264d947f2dc8c3bbc276f58f868/src/posix/platform/uart.cpp#L165-L168

Reply:
> me of the COI output depends on how many children you have attached.

Yes, agree that 8KB is fine for POSIX.

Reply:
I tend to use less memory, openwrt is also posix, which has very limited resource.

Reply:
> I tend to use less memory, openwrt is also posix, which has very limited resource.

OpenWrt platforms have MBs of memory, right? Several kB is pretty small in comparison.

I don't think even 2kB is enough to print a full router table. At 8kB, the current child table CLI output is limited to somewhere around 100 children. We have already seen deployments with routers that support more than 100 children.